packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -21,9 +24,13 @@ type Props = {
 const ConnectorRow = ({ type, connectors, onClickSetup }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const firstConnector = connectors[0];
+  const firstStandardConnector = connectors.find(({ isStandard }) => isStandard);
```

Comment:
instead of finding the first one, will it be better to assert there's only one connector and standard connector in the list if needed?

Reply:
or join the names with `, `

Reply:
For multi-platform case, there will be more than one connector.

Reply:
Updated: assert there's only one standard connector because there is no reason to have multiple for now.